packages/liveblocks-client/src/immutable.ts

```diff
@@ -32,7 +32,7 @@ function liveListToJson(value: LiveList<any>) {
   return value.toArray().map(liveNodeToJson);
 }
 
-export function liveNodeToJson(value: any): any {
+export function liveNodeToJson(value: unknown): any {
```

Comment:
Here, `unknown` is used intentionally. The point of this function is to take any value and as such its type is unknown (but it won't disable typechecking inside this function, like `any` does!).